src/styles/index.scss

```diff
@@ -1,3 +1,60 @@
+@import 'variables';
+
 body {
   margin: 0;
+  height: 100vh;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  color: $black-color;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $border-day-gap;
+
+  padding: 10px;
+  width: ($day-size * 7 + $border-day-gap * 6);
```

Comment:
The variable `$day-size` should be defined in the `variables.scss` file. Ensure that it is correctly imported and defined there.